### DISCUSSION TITLE
Remove 1-hit KO

### DISCUSSION THREAD

**Discussion by rglezg** [Renegade-Engine/Renegade-Emerald#13] [Ideas]
Continuation of #11 
Not much to say here. 1-hit KO moves feel broken in some sense. Should we keep them as they are, nerfe them, or remove them?

**Comment by rglezg** [comment]
NOTE: Here we're talking about Guillotine, Sheer Cold, Fissure and Horn Drill. While Perish Song and Destiny Bound can be called 1-hit KO too, they are fine.

**Comment by DP-404** [comment]
The 1-hit KO movements can be modified according to the next points in order to make them less overpowered.

- Set all movements PP to 1, this way they can only be used once per battle.
- Define a low standard accuracy. 

- When using the movement, there will be different outcomes according to the movement type and the target's:
1. If the movement is effective against the target > the movement accuracy rises a bit.
2. If the movement is not very effective > the movements fails.
3. If none of the previous conditions are met > the movement is used without any modifications.

**Reply by rglezg** [reply]
> Set all movements PP to 1, this way they can only be used once per battle.

Having that no Pokémon can learn more than 4 moves, I think that would make all 1-hit KO nearly useless. Points 2 and 3 seem interesting and I'd give them a chance.

**Comment by SoyNegro** [comment]
The problems with 1-Hit KO, to my understanding are basically two. One Mind Reader + Any 1-Hit KO = Bored Battle. Two if nerfed too much, since they already have low accuracy and need to meet others parameters to work plus only 5pp which will be brought down to 1 or 2 by planned features, they wouldn't be worthy anymore. Anyways if we choose to kept them, I suggest banning those moves from battles against trainers.

**Reply by rglezg** [reply]
> One Mind Reader + Any 1-Hit KO = Bored Battle.

AFAIK, only two Pokémon can do that: Articuno (Mind Reader + Sheer Cold), and Smeargle. Articuno's learnset can be modiffied to remove Mind Reader, and Smeargle's Sketch can be tweaked to fail if you try to copy a 1-hit KO.
